iOS Example Frame/iOS Example Frame/CustomStyle2.swift

```diff
@@ -31,6 +31,7 @@ struct PaymentFormStyleCustom2: PaymentFormStyle {
     var editBillingSummary: BillingSummaryViewStyle? = EditBillingSummaryStyleCustom2()
     var cardNumber: CellTextFieldStyle?
     var expiryDate: CellTextFieldStyle? = ExpiryDateFormStyleCustom2()
+    var securityCode: CellTextFieldStyle? =  SecurityCodeFormStyleCustom2()
```

Comment:
You have quite a few places where the spacing is very inconsistent (2 spaces, or space either side of `:`, small things like this that will end up in another PR later.

Can you please run a SwiftLint autocorrect to have it all cleaned?

Reply:
Swiftlint is not supported yet

Reply:
I know this will be annoying, but would you please mind reviewing [SwiftLint update PR](https://github.com/checkout/frames-ios/pull/202/files). If we merge that in quickly (I've self reviewed every line changed and I'm happy its only syntax fixing) and pull that in your branch, you can easily fix all the inconsistencies at once via the terminal 🙏